### PR TITLE
Global Hub known issue for 2.10 (don't merge, get issue, rebuild/edit in VS code)

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

**Comment by clyang82** [comment]
/lgtm for me for the content.

assign @jc-berger for help on revising the description. Thanks.

**Comment by swopebe** [comment]
@yanmxa TY for alerting us to this. These should come to us via a JIRA issue, PRs like this are difficult to rebuild in the browser. Please see the process for opening issues for the Doc team here:

https://docs.google.com/document/d/1YTqpZRH54Bnn4WJ2nZmjaCoiRtqmrc2w6DdQxe_yLZ8/edit#heading=h.9fvyr2rdriby

Basically on your bug, **open a doc issue following that process** (a task for us) and fill out the template. PRs are great for smaller changes that don't impact our doc build. 

I am going to ask Jake when he returns to use this PR as a draft to create the known issue. Please attach the doc issue to this PR for me and I will make sure he sees it.

This is an example of an issue requesting a "known issue": https://issues.redhat.com/browse/ACM-10292

File: release_notes/known_issues_global.adoc [hunk 1]
```diff
@@ -47,4 +47,67 @@ This is also an issue for the **Cluster Group Compliancy Overview**.
 
 If a managed hub cluster imports an {ocp-short} 3.11 cluster (deprecated) as managed cluster, it cannot redirect to the _Observability_ page in the *Global Hub* > *Overview* dashboard.
 
-You need to manually navigate to your target page.
\ No newline at end of file
+You need to manually navigate to your target page.
+
+[#compliance-job-failed-with-reimported-hub]
+== Compliance cronjob might fail when reimport the managed hub with a different name
```
Comment (swopebe) [inline release_notes/known_issues_global.adoc:53]: This needs to be rebuilt and edited quite a bit, which may be easier in a new branch and PR by the writer in VS Code.

**Comment by yanmxa** [comment]
> @yanmxa TY for alerting us to this. These should come to us via a JIRA issue, PRs like this are difficult to rebuild in the browser. Please see the process for opening issues for the Doc team here:
> 
> https://docs.google.com/document/d/1YTqpZRH54Bnn4WJ2nZmjaCoiRtqmrc2w6DdQxe_yLZ8/edit#heading=h.9fvyr2rdriby
> 
> Basically on your bug, **open a doc issue following that process** (a task for us) and fill out the template. PRs are great for smaller changes that don't impact our doc build.
> 
> I am going to ask Jake when he returns to use this PR as a draft to create the known issue. Please attach the doc issue to this PR for me and I will make sure he sees it.
> 
> This is an example of an issue requesting a "known issue": https://issues.redhat.com/browse/ACM-10292

@swopebe Thanks for your clarification! I just opened a Jira to trace this known issue for release 2.10. 

The Jira task: https://issues.redhat.com/browse/ACM-10343

**Comment by jc-berger** [comment]
Closing this PR since these doc concerns have been addressed in my PR here: https://github.com/stolostron/rhacm-docs/pull/6225/files

**Pull request closed without merging**